scripts/lib.sh

```diff
@@ -12,6 +12,7 @@ GO_RUNTIME_PKG=${GO_RUNTIME_PKG:-github.com/kata-containers/runtime}
 # Give preference to variable set by CI
 KATA_BRANCH=${branch:-}
 KATA_BRANCH=${KATA_BRANCH:-master}
+yq_file="${script_dir}/../scripts/install-yq.sh"
```

Comment:
seems that this path doesn't exist.

Reply:
@chavafg : Fixed the file path issue. However mounting error appears to be coming from the master branch.
```
docker: Error response from daemon: OCI runtime create failed: rpc error: code = Internal desc = Could not create destination mount point: /etc/resolv.conf: open /etc/resolv.conf: no such file or directory: unknown.
INFO: ERROR: test failed
```